website/content/docs/networking/index.mdx

```diff
@@ -38,3 +38,15 @@ only the high-level networking options until you are comfortable
 with the Vagrant workflow and have things working at a basic level.
 Provider-specific network configuration can very quickly lock you out
 of your guest machine if improperly done.
+
+
+## Networking Assumptions
```

Comment:
I couldn't think of other assumptions to add here

Reply:
Please note the typo: Virutalbox